### PR TITLE
aarch64:update checkdev judging

### PR DESCRIPTION
**Pull request by zhenyzha**
aarch64:update checkdev judging

ID: 2020550
Signed-off-by: Zhenyu Zhang <zhenyzha@redhat.com>

### CODE REVIEW

**Comment by zhenyzha** [comment]
```
 (01/10) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.aarch64.io-github-autotest-qemu.virtio_port_hotplug.hotplug_port_pci.arm64-pci: PASS (67.44 s)
 (02/10) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.aarch64.io-github-autotest-qemu.virtio_port_hotplug.hotplug_port_pci.with_reboot.arm64-pci: PASS (167.18 s)
 (03/10) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.aarch64.io-github-autotest-qemu.virtio_port_hotplug.hotplug_port_pci.with_system_reset.arm64-pci: PASS (152.22 s)
 (04/10) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.aarch64.io-github-autotest-qemu.virtio_port_hotplug.hotplug_port_pci.with_shutdown_after_unplug.arm64-pci: PASS (76.72 s)
 (05/10) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.aarch64.io-github-autotest-qemu.virtio_port_hotplug.hotplug_port_pci.with_shutdown_after_plug.arm64-pci: PASS (64.62 s)
 (06/10) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.aarch64.io-github-autotest-qemu.virtio_port_hotplug.hotplug_port_pci.with_live_migration_after_unplug.arm64-pci: PASS (96.38 s)
 (07/10) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.aarch64.io-github-autotest-qemu.virtio_port_hotplug.hotplug_port_pci.repeat_pci_in_loop.arm64-pci: PASS (76.61 s)
 (08/10) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.aarch64.io-github-autotest-qemu.virtio_port_hotplug.hotplug_port_pci.without_chardev.arm64-pci: PASS (50.28 s)
 (09/10) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.aarch64.io-github-autotest-qemu.virtio_port_hotplug.hotplug_port_pci.with_console_negative.arm64-pci: PASS (68.46 s)
 (10/10) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.aarch64.io-github-autotest-qemu.virtio_port_hotplug.hotplug_port_pci.max_chardevs.arm64-pci: PASS (356.69 s)
```
@PaulYuuu Could you pls help review, when you are free. thanks.

File: qemu/tests/virtio_serial_hotplug_max_chardevs.py [hunk 1]
```diff
@@ -107,8 +107,8 @@ def run_bg_test():
     try:
         char_devices[0].hotplug(vm.monitor, vm.devices.qemu_version)
     except QMPCmdError as e:
-        if "attempt to add duplicate property '%s'" % char_devices[0] \
-                not in str(e.data):
+        if not ("duplicate property '%s'" % char_devices[0] in str(e.data)
+                or "'%s' already exists" % char_devices[0] in str(e.data)):
```
Comment (PaulYuuu) [inline qemu/tests/virtio_serial_hotplug_max_chardevs.py:111]: Hello @zhenyzha, is it(`"attempt to add duplicate property '%s'"` --> `"duplicate property '%s'"`) sure that it won't have any effect?

Comment (zhenyzha) [reply qemu/tests/virtio_serial_hotplug_max_chardevs.py:111]: In fact, that's a piece of negative test information, as long as it triggered an exception and correctly displays the reason, I think it has no effect.

Comment (PaulYuuu) [reply qemu/tests/virtio_serial_hotplug_max_chardevs.py:111]: OK, thanks.

**Pull request merged**